ImageSaver/app/src/main/java/com/example/jong/imagesaver/saved_images.java

```diff
@@ -0,0 +1,173 @@
+package com.example.jong.imagesaver;
+
+import android.app.Activity;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.net.Uri;
+import android.os.Bundle;
+import android.app.Fragment;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.widget.Button;
+import android.widget.ImageButton;
+
+import com.example.jong.imagesaver.db.Dbsetup;
+import com.example.jong.imagesaver.db.FeedReaderDbHelper;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+
+public class saved_images extends Fragment {
+
+    private OnFragmentInteractionListener mListener;
+    private WebView webView;
+    int position=1;
+    FeedReaderDbHelper mDbHelper = MainActivity.mDbHelper;
+    SQLiteDatabase readabledb = mDbHelper.getReadableDatabase();
+
+
+     public static saved_images newInstance(String param1, String param2) {
+        saved_images fragment = new saved_images();
+        Bundle args = new Bundle();
+         fragment.setArguments(args);
+        return fragment;
+    }
+
+    public saved_images() {
+        // Required empty public constructor
```

Comment:
the method being empty could cause random issues, you should at least do something so that you know if/when the method is called.